client/tasks/tasks.tsx

```diff
@@ -116,6 +116,10 @@ export const Tasks: React.FC< TasksProps > = ( { query } ) => {
 			tasks,
 		} = taskList;
 
+		if ( isHidden ) {
```

Comment:
Nice.  Not directly related, but I find it interesting our tasks are "visible" but our task list is "hidden."  Not very important but I was thinking of inverting the list and using `isVisible` (keeping `isHidden` for backwards compatibility in the API).  Any thoughts on this?

Reply:
Yeah, makes sense to me to be consistent. Made this change in https://github.com/woocommerce/woocommerce-admin/pull/7733/commits/06fdc31a4ef6ad62a3b9d8ad21f13464a078ab87 ...in a bit of a hurry, so hopefully all is well!